resources/scripts/api/account/getTwoFactorTokenData.ts

```diff
@@ -1,13 +1,14 @@
-import http from "@/api/http";
+import http from '@/api/http';
 
 export interface TwoFactorTokenData {
+    /* eslint-disable camelcase */
     image_url_data: string;
     secret: string;
```

Comment:
Forgot to add `/* eslint-enable camelcase */`

Reply:
confused, what do you achieve by doing do? Shouldn't `eslint-disable` be enough? Why enable and disable it for the same line?

Reply:
`eslint-disable` disables it for the rest of the file, the `eslint-enable` reenables the rule.  This is preferred as if anyone adds another field to that interface with a camelcase, it is allowed without them having to fiddle with the rules.

Reply:
Why not just use  `eslint-disable-next-line` then?

Reply:
> Why not just use `eslint-disable-next-line` then?

You can, but if there is any other fields added you would then have to change the eslint comment.  Or you can just have the interface use `imageUrlData` and change the request callback to `then(({ data }) => resolve({ imageUrlData: data.data.image_url_data, secret: secret }))` which is probably the better approach as that interface will be passed around and then you would need an eslint-ignore anytime you read `image_url_data`.

Reply:
You only need the ignore when defining the object, not when accessing it. Eslint is smart.